src/shared/dom.js

```diff
@@ -238,4 +238,8 @@ export function addResizeListener(element, fn) {
 			element.removeChild(object);
 		}
 	};
-}
\ No newline at end of file
+}
+
+export function toggleClass(element, name, toggle) {
+	element.classList.toggle(name, Boolean(toggle));
```

Comment:
Do we need the `Boolean` here? In Chrome at least it coerce the value for us

Reply:
Update: tested in Chrome, Firefox and Safari. I reckon we can do without — I'll delete it